war/src/main/js/tooltips.js

```diff
@@ -0,0 +1,92 @@
+import tippy from "tippy.js"
+
+const TOOLTIP_BASE = {
+  arrow: false,
+  theme: "tooltip",
+  animation: "tooltip",
+  appendTo: document.body
+}
+
+let tooltipInstances = []
+const globalPlugin = {
+  fn(instance) {
+    return {
+      onCreate() {
+        tooltipInstances = tooltipInstances.concat(instance)
+      },
+      onDestroy() {
+        tooltipInstances = tooltipInstances.filter(i => i !== instance)
+      }
+    }
+  }
+}
+
+tippy.setDefaultProps({
+  plugins: [globalPlugin]
+})
+
+registerTooltips()
+
+/**
+ * Registers tooltips for the page
+ * If called again, destroys existing tooltips and registers them again (useful for progressive rendering)
+ * @param {HTMLElement} container - Registers the tooltips for the given container
+ */
+function registerTooltips(container) {
+  if (!container) {
+    container = document
+  }
+
+  tooltipInstances.forEach(instance => {
+    if (instance.props.container === container) {
+      instance.destroy()
+    }
+  })
+
+  tippy(container.querySelectorAll("[tooltip]:not([tooltip=\"\"])"), Object.assign({
+    content: element => element.getAttribute("tooltip")
+      .replace("<br>", "\n")
+      .replace("<br/>", "\n")
```

Comment:
I think this needs to be done for html-tooltip too, 

<img width="371" alt="image" src="https://user-images.githubusercontent.com/21194782/166323041-4e7ece4c-41c3-4359-8585-d8ee53204a1d.png">

br doesn't seem to work there currently

Reply:
Actually it's just that the replace is only applying to the first match.

Needs `replaceAll`

Reply:
Fixed in https://github.com/jenkinsci/jenkins/pull/6408/commits/e409234fb1a3c3eb7e792ad26c6d3c006a80cab4